src/sqlparser/src/ast/mod.rs

```diff
@@ -1109,12 +1111,17 @@ impl fmt::Display for Statement {
                 table_name,
                 columns,
                 source,
+                returning,
             } => {
                 write!(f, "INSERT INTO {table_name} ", table_name = table_name,)?;
                 if !columns.is_empty() {
                     write!(f, "({}) ", display_comma_separated(columns))?;
                 }
-                write!(f, "{}", source)
+                write!(f, "{}", source)?;
+                if !returning.is_empty() {
+                    write!(f, " RETURNING {:?}", returning)?;
```

Comment:
This seems incorrect. 🤔 It will print the `Vec` in the Rust style instead of comma-separated list.

Reply:
I'm not sure about what is this code for? Where is the result written to?

Reply:
This is for unparsing the AST into the SQL. With `ast.to_string()` or `format!("{ast}")`, you can get a formatted SQL.

Reply:
please check the current implementation